ci/Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+                            'Linux Nnvidia Dbg32': runJob.gfxrTestLinux('Linux Nvidia Dbg32', runJob.ReleaseMode, runJob.LinuxNvidiaLabel, runJob.Bit32, "commit-suite.json"),
+                            'Mac': runJob.gfxrTestLinux('Mac', runJob.ReleaseMode, runJob.MacLabel, runJob.Bit64, "commit-suite.json"),
+                            'Windows AMD Rel64': runJob.gfxrTestWindows('Windows AMD Rel64', runJob.ReleaseMode, runJob.WinAMDLabel, runJob.Bit64, "commit-suite.json"),
+                            'Windows AMD Dbg32': runJob.gfxrTestWindows('Windows AMD Dbg32', runJob.ReleaseMode, runJob.WinAMDLabel, runJob.Bit32, "commit-suite.json"),
+                            'Windows Nvidia Rel64': runJob.gfxrTestWindows('Windows Nvidia Rel64', runJob.ReleaseMode, runJob.WinNvidiaLabel, runJob.Bit64, "commit-suite.json"),
+                            'Windows Nvidia Dbg32': runJob.gfxrTestWindows('Windows Nvidia Dbg32', runJob.ReleaseMode, runJob.WinNvidiaLabel, runJob.Bit32, "commit-suite.json"),
+                            'Windows11 AMD Rel64': runJob.gfxrTestWindows('Windows11 AMD Rel64', runJob.ReleaseMode, runJob.Win11AMDLabel, runJob.Bit64, "commit-suite.json"),
+                            'Windows11 AMD Dbg32': runJob.gfxrTestWindows('Windows11 AMD Dbg32', runJob.ReleaseMode, runJob.Win11AMDLabel, runJob.Bit32, "commit-suite.json"),
+                            'Windows11 ARM Rel64': runJob.gfxrTestWindows('Windows11 ARM Rel64', runJob.ReleaseMode, runJob.Win11ARMLabel, runJob.Bit64, "commit-suite.json"),
+                            'Windows11 ARM Dbg64': runJob.gfxrTestWindows('Windows11 ARM Dbg64', runJob.ReleaseMode, runJob.Win11ARMLabel, runJob.Bit64, "commit-suite.json")
+                        ]
+                    }
+                    // parallel runs OUTSIDE the node block - each branch allocates its own node
+                    parallel tests
+                }
+            }
+        }
+
+        stage('Trigger Extended Tests') {
+            when {
+                branch 'dev'
+            }
+
+            steps {
+                script {
+                    def quietPeriodSeconds = calculateQuietPeriod()
+                    build job: 'beau-lunarg/gfxreconstruct-pipeline-dev-extended',
+                          wait: false,
+                          propagate: false,
+                          quietPeriod: quietPeriodSeconds
+                }
+            }
+        }
+    }
+}
+
+def calculateQuietPeriod() {
+    def now = new Date()
+    def hour = now.getHours()
+    def dayOfWeek = now.getDay()  // 0=Sun, 1=Mon, ..., 6=Sat
+
+    // In window (Mon-Fri 8pm-midnight): run immediately
+    if (dayOfWeek >= 1 && dayOfWeek <= 5 && hour >= 20 && hour < 24) {
+        return 0
+    }
+
+    // Calculate seconds until next 8pm Mon-Fri
+    def target = now.clone()
+    target.setHours(20)
+    target.setMinutes(0)
+    target.setSeconds(0)

```

Comment:
Acceptable, but a little odd given that we reset `Seconds` to 0 but not milliseconds.